### DISCUSSION TITLE
Item api lookup based on tmdb id

### DISCUSSION THREAD

**Discussion by aleksilassila** [jellyfin/jellyfin#10156] [Q&A] [answered]
Hey,

I was wondering if there's a way of retrieving a jellyfin item or an id from the jellyfin api based on a provider id such as tmdb id. I didn't see an option for this in the [item search route](https://api.jellyfin.org/#tag/Items/operation/GetItems) in jellyfin API specification.

Cheers!

**Comment by nielsvanvelzen** [comment, marked as answer]
It is not possible to query items by provider is. Some endpoints do support filtering if a certain provider is set but that's all there is.

**Reply by aleksilassila** [reply]
What endpoints exactly? Doesn't that achieve the same thing as a lookup in practice

**Reply by nielsvanvelzen** [reply]
With the endpoint you linked you can do that, use `hasTmdbId=true` and it will filter the results to only items with a tmdb id.
